File: manager-web/src/main/webapp/js/service/paymentMethodService.js
//服务层
app.service('paymentMethodService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../paymentMethod/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../paymentMethod/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../paymentMethod/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../paymentMethod/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../paymentMethod/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../paymentMethod/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../paymentMethod/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: content-interface/src/main/java/com/service/PaymentMethodService.java
package com.service;

import com.shopxx.pojo.XxPaymentMethod;
import entity.PageResult;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface PaymentMethodService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<XxPaymentMethod> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(XxPaymentMethod paymentMethod);
	
	
	/**
	 * 修改
	 */
	public void update(XxPaymentMethod paymentMethod);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public XxPaymentMethod findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(XxPaymentMethod paymentMethod, int pageNum, int pageSize);
	
}
